src/js/components/Product.js

```diff
@@ -166,11 +203,12 @@ class Product {
       },
     });
 
-    thisProduct.dom.element.dispatchEvent(event);
+    thisProduct.element.dispatchEvent(event);
```

Comment:
Dlaczego to zmieniłaś? Przez tą zmianę nie działa dodawanie produktów do koszyka. Powinno być, tak jak było przedtem:
```suggestion
    thisProduct.dom.element.dispatchEvent(event);
```